src/deriv_api/SubscriptionManager.js

```diff
@@ -2,6 +2,8 @@ import {
     first, finalize, share,
 } from 'rxjs/operators';
 
+import { APIError }                       from './errors';
+
 import { objectToCacheKey }               from './utils';
```

Comment:
nitpick: whitespaces

Reply:
We don't use prettier here. I tried to remove whitespaces but es-lint forces to add spaces until it is aligned